static/js/components/PhotometryPlot.jsx

```diff
@@ -851,7 +895,7 @@ const PhotometryPlot = ({
       setLayouts(newLayouts);
       setInitialized(true);
     }
-  }, [photometry, dm, filter2color, defaultVisibleFilters]);
```

Comment:
Do you still want filter2color since it's in the useEffect?

Reply:
Good point. Just added it back